core/src/main/scala/kafka/admin/BrokerApiVersionsCommand.scala

```diff
@@ -17,11 +17,14 @@
 
 package kafka.admin
 
+
+
```

Comment:
What's these for?

Reply:
Sorry, these are not needed at all, it was me moving the `import joptsimple.OptionSpec` in the right place. I'll remove the spaces